### DISCUSSION TITLE
Transcript (SRT) to Summary Possible?

### DISCUSSION THREAD

**Discussion by natesct** [deepgram/community#679] [General help] [answered]
Hi

I have thousands of transcripts in SRT format, is it possible to send a SRT via the API and return the summary? It would be too costly to redo the audio to text process. They were done in bulk with another service provider.

Regards

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @natesct, we do offer a text-input API for summarizing transcripts rather than audio! Please consult our text summarization doc: https://developers.deepgram.com/docs/text-summarization

**Reply by natesct** [reply]
Thank you very much.
